main.go

```diff
@@ -135,8 +134,6 @@ func main() {
 		LeaderElection:          enableLeaderElection,
 		LeaderElectionID:        "ab76f4c9.openshift.io",
 		LeaderElectionNamespace: operatorNamespace,
-		Namespace:               operatorNamespace,
-		Cache:                   cache.Options{Namespaces: []string{operatorNamespace}},
```

Comment:
@iamniting This change has many additional implications besides watching all namespaces. We solved these kinds of issue using a secondary client in other codebases

Reply:
Hi Ohad i have made the changes as discussed